wordpress-mobile/gutenberg-mobile
# [Android] setSelection for Aztec wrapper

Fixes #602 

~This PR introduces a "setSelection" functionality to the Aztec wrapper component. It is currently wired up to the `RichText` component as a prop and updated every time the locally kept `selectionStart` and `selectionEnd` changes. See https://github.com/WordPress/gutenberg/pull/13978.

With this setting of selection, the caret position is set to 0 for the newly created block from the block split.~

This PR adds a `MovementMethod` that places the caret at the beginning of text when focusing the view. The Android default is to place is at the end, but in our case, placing it at the start is more appropriate.

To test:

1. In a paragraph block that has some text, place the caret in the middle of the text
2. Tap "Enter" to split the block
3. Notice the caret being placed at the beginning of the new block
